### ISSUE TITLE
Add support for Firestore v11

### ISSUE THREAD

**Issue by rtpHarry** [FirebaseExtended/rxfire#117]
Tried a normal install and a canary install but it doesn't work:

```shell
❯ npm i rxfire@canary
npm error code ERESOLVE
npm error ERESOLVE unable to resolve dependency tree
npm error
npm error While resolving: angularstart-chat@0.0.0
npm error Found: firebase@11.0.1
npm error node_modules/firebase
npm error   firebase@"^11.0.1" from the root project
npm error
npm error Could not resolve dependency:
npm error peer firebase@"^9.0.0 || ^10.0.0" from rxfire@6.0.6-canary.5cfad21
npm error node_modules/rxfire
npm error   rxfire@"6.0.6-canary.5cfad21" from the root project
```

**Comment by quedicesebas** [comment]
Anyone?

**Comment by jamesdaniels** [comment]
Fixed in v6.1
